internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -45,13 +45,26 @@ username: %s
 password: %s
 `
 
+const quickstartTemplateIntro = `You are creating a new Atlas cluster and enabling access to it.
+
+Press [Enter] to use the default values.
+
+Enter [?] on any option to get help.
+`
+
+const quickstartTemplateCluster = `
+Creating your cluster... [It's safe to 'Ctrl + C']
+`
+
 const (
 	replicaSet      = "REPLICASET"
 	atlasM2         = "M2"
 	atlasAdmin      = "atlasAdmin"
 	mongoshURL      = "https://www.mongodb.com/try/download/shell"
 	atlasAccountURL = "https://docs.atlas.mongodb.com/tutorial/create-atlas-account/?utm_campaign=atlas_quickstart&utm_source=mongocli&utm_medium=product/"
 	profileDocURL   = "https://docs.mongodb.com/mongocli/stable/configure/?utm_campaign=atlas_quickstart&utm_source=mongocli&utm_medium=product#std-label-mcli-configure"
+	aws             = "AWS"
+	region          = "US_EAST_1"
```

Comment:
👀 Is this configurable for the user? Push european customers to use a US cluster might not go down so well...

Reply:
yes it is, they have to use `--tier and --region`. This is non configurable if the use `--default` for now but I see that one of the Gus comment is about to enable `--default` with also other flags.

Reply:
Yup combining flags will help and resolve this point for me as well. Thanks for the clarification.